pkg/controller/backup_controller.go

```diff
@@ -942,7 +963,16 @@ func (c *backupController) checkVolumeSnapshotReadyToUse(ctx context.Context, vo
 			return err
 		})
 	}
-	return eg.Wait()
+
+	err := eg.Wait()
```

Comment:
Check error?

Reply:
Error is needed for return and record. No need for handling.